pom.xml

```diff
@@ -0,0 +1,208 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>Services</groupId>
```

Comment:
Let's do edu.ucla.library for the groupId. I believe this will be the first thing we've pushed into Maven Central from the library and it will be best, I think, to setup the workflow so that others in the library who want to publish Jars can also take advantage of the same group ID.